aiocache/backends/memory.py

```diff
@@ -131,3 +135,6 @@ def __init__(self, serializer=None, **kwargs):
     @classmethod
     def parse_uri_path(cls, path):
         return {}
+
+    def build_key(self, key: str, namespace: Optional[str] = None) -> str:
+        return self._str_build_key(key, namespace)
```

Comment:
You might, however, be able to just do:
`build_key = _str_build_key`
Rather than trying to set it in the `__init__()` method.

Reply:
I'm having no luck with that approach.